lib/pages/home.dart

```diff
@@ -30,6 +29,14 @@ import '../widgets/trufi_drawer.dart';
 
 class HomePage extends StatefulWidget {
   static const String route = '/';
+  final LocaleWidgetBuilder customOverlayWidget;
+  final WidgetBuilder customBetweenFabWidget;
+
+  const HomePage(
+      {Key key,
+      @required this.customOverlayWidget,
```

Comment:
I am not sure if they should be required or optional.

Reply:
Optional